frequency.py

```diff
@@ -2,12 +2,12 @@
 from collections import defaultdict
 
 order = [
-	["subspecies", "varietas", "forma"],
+	["subspecies", "strain", "varietas", "forma"],
```

Comment:
@ninewise Is this the correct place to add "strain" in this file? What is this file used for?

Reply:
This script is used to show the frequency of ranks in an aggregated index. I used it to evaluate other aggregation methods on the backend for the sequences table. Eventually I replaced the LCA* aggregation for the metagenomics backend with a 95% LCA* hybrid approach. We haven't evaluated the use to this for the metaproteomics backend, yet.

And yes, this is the correct place to add 'strain'.